src/main/java/frc/robot/RobotContainer.java

```diff
@@ -100,52 +137,151 @@ public static RobotContainer getInstance() {
+   * ({@link edu.wpi.first.wpilibj.Joystick} or {@link XboxController}), and 
+   * Then passing it to a
    * {@link edu.wpi.first.wpilibj2.command.button.JoystickButton}.
    */
   private void configureButtonBindings() {
     // BEGIN AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=BUTTONS
-// Create some buttons
-final JoystickButton halfSpeedBtn = new JoystickButton(rightJoystick, 7);        
-halfSpeedBtn.onTrue(new HalfSpeed( m_drivetrain ).withInterruptBehavior(InterruptionBehavior.kCancelSelf));
-                        
-final JoystickButton quarterSpeedBtn = new JoystickButton(rightJoystick, 6);        
-quarterSpeedBtn.onTrue(new QuarterSpeed( m_drivetrain ).withInterruptBehavior(InterruptionBehavior.kCancelSelf));
-                        
-final JoystickButton reverseDriveBtn = new JoystickButton(rightJoystick, 1);        
-reverseDriveBtn.onTrue(new ReverseDrive( m_drivetrain ).withInterruptBehavior(InterruptionBehavior.kCancelSelf));
-                        
-final JoystickButton closeClawBtn = new JoystickButton(rightJoystick, 12);        
-closeClawBtn.onTrue(new CloseClaw( m_claw ).withInterruptBehavior(InterruptionBehavior.kCancelSelf));
-                        
-final JoystickButton openClawBtn = new JoystickButton(rightJoystick, 10);        
-openClawBtn.onTrue(new OpenClaw( m_claw ).withInterruptBehavior(InterruptionBehavior.kCancelSelf));
-                        
-final JoystickButton shiftLowBtn = new JoystickButton(leftJoystick, 12);        
-shiftLowBtn.onTrue(new ShiftLow( m_drivetrain ).withInterruptBehavior(InterruptionBehavior.kCancelSelf));
-                        
-final JoystickButton shiftHighBtn = new JoystickButton(leftJoystick, 10);        
-shiftHighBtn.onTrue(new ShiftHigh( m_drivetrain ).withInterruptBehavior(InterruptionBehavior.kCancelSelf));
-                        
-
+    // Create some buttons
+    final JoystickButton halfSpeedBtn = new JoystickButton(rightJoystick, 1);
+    halfSpeedBtn.onTrue(new HalfSpeed(m_drivetrain).withInterruptBehavior(InterruptionBehavior.kCancelSelf));
+
+    // final JoystickButton quarterSpeedBtn = new JoystickButton(rightJoystick, 6);
+    // quarterSpeedBtn.onTrue(new QuarterSpeed(m_drivetrain).withInterruptBehavior(InterruptionBehavior.kCancelSelf));
+
+    final JoystickButton reverseDriveBtn = new JoystickButton(leftJoystick, 1);
+    reverseDriveBtn.onTrue(new ReverseDrive(m_drivetrain).withInterruptBehavior(InterruptionBehavior.kCancelSelf));
+
+    // final JoystickButton turnNDegrees90Btn = new JoystickButton(rightJoystick, 10);
+    // turnNDegrees90Btn
+    //     .onTrue(new TurnNRelative(m_drivetrain, 90).withInterruptBehavior(InterruptionBehavior.kCancelSelf));
+
+    // final JoystickButton turnNDegrees180Btn = new JoystickButton(rightJoystick, 12);
+    // turnNDegrees180Btn
+    //     .onTrue(new TurnNRelative(m_drivetrain, 180).withInterruptBehavior(InterruptionBehavior.kCancelSelf));
```

Comment:
Same with all these. If things aren't needed from robotbuilder, remove them from robotbuilder instead of commenting them.